plans/tier1.fmf

```diff
@@ -228,9 +228,20 @@
     playbook: tests/ansible_collections/roles/reboot/main.yml
 
 /convert_offline_systems:
+  # At the moment we have only CentOS7 repos available on Satellite server
+  adjust:
+    enabled: false
+    when: >
+      distro != centos-7
   discover+:
     test: checks-after-conversion
   prepare+:
+  - name: workaround_kernel
+    how: shell
+    script: "yum install kernel-3.10.0-1160.76.1.el7 -y && yum remove kernel-3.10.0-1160.80.1.el7 -y"
+  - name: reboot machine
+    how: ansible
+    playbook: tests/ansible_collections/roles/reboot/main.yml
```

Comment:
Do we need to reboot?
Wouldn't `grubby --set-default-kernel` be sufficient?
I would probably add some comment about the kernel workaround being temporary.
Or a `#TODO` to remove it, when the repository gets synced over at Satellite.
But I guess for the sake of saving some time, we can also keep it as is.

Reply:
The reboot would be needed I believe as the c2r check is done against the running kernel which changes only after reboot.